before-commit.sh

```diff
@@ -44,6 +44,15 @@ for binary in "${binaries[@]}"; do
 	fi
 done
 
+echo -e "${GREEN} Building charts test${NC}"
+${buildEnv} go test -v -c -o hb_chart_test ./test/charts/helm_broker_test.go
+goBuildResult=$?
+if [[ ${goBuildResult} != 0 ]]; then
+    echo -e "${RED}✗ go test -c ${binary} ${NC}\n$goBuildResult${NC}"
```

Comment:
`binary` variable comes from the previous loop (`for binary in "${binaries[@]}"; do (...) done`) so the result of `echo` is `go build targz` instead e.g. `go build hb_chart_test` (or just `go build` if we run this part separately)